api/app/email_template/repository.py

```diff
@@ -0,0 +1,9 @@
+from app import db
+
+from app.email_template.models import EmailTemplate
+
+class EmailRepository():
+
+    @staticmethod
+    def get(event_id, key):
+        return db.session.query(EmailTemplate).filter_by(event_id=event_id, key=key).first()
```

Comment:
Think it's best to implement the fall-back logic here

Reply:
Done